pkg/proto/transactions.go

```diff
@@ -458,7 +458,7 @@ func (tx *Genesis) generateID(scheme Scheme) error {
 	return nil
 }
 
-func (tx Genesis) GetID(scheme Scheme) ([]byte, error) {
+func (tx *Genesis) GetID(scheme Scheme) ([]byte, error) {
```

Comment:
Is it OK to have a mixture of different receivers? May be better to make all of them pointers?

Reply:
I'll do it in new PR.